airbyte-webapp/src/components/connectorBuilder/types.ts

```diff
@@ -636,7 +636,7 @@ export const convertToManifest = (values: BuilderFormValues): ConnectorManifest
 
   const spec: Spec = {
     connection_specification: specSchema,
-    documentation_url: "",
+    documentation_url: "https://docsurl.com",
```

Comment:
nit: What about using `https://example.org/` for this? Its official purpose is to be used as an example

Reply:
I like that suggestion, will update